### PR TITLE
Derek

### PR DESCRIPTION
**Pull request by djsaugatuck**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,6 +30,27 @@
...
+        <div class="dropdown">
+          <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          NFL Teams
+          </button>
+          <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+          <!--<a class="dropdown-item" href="#">Action</a> example here -->  
+        </div>
+      </div>
+    </div>
+  </body>
```
Comment (ghost) [inline index.html:50]: Careful not to close the body out here, since it needs to wrap the entire page contents.

File: index.html [hunk 2]
```diff
@@ -30,6 +30,27 @@
...
+
+<body>
```
Comment (ghost) [inline index.html:52]: Same as above, you probably want to delete this opening body tag, and let the whole page be wrapped in a single body element.

**Review by ghost — CHANGES_REQUESTED** [review]
See per-line comments

**Comment by djsaugatuck** [comment]
removing unneeded <body>

**Pull request closed without merging**